edit.go

```diff
@@ -108,15 +110,24 @@ func edit(name string, v *vaulted.Vault) {
 		printVariables(v)
 		printAWS(v, false)
 		printSSHKeys(v)
+		printDuration(v)
 
-		input := readMenu("\nEdit vault: [a,s,v,?,q]: ")
+		input := readMenu("\nEdit vault: [a,s,v,d,?,q]: ")
 		switch input {
 		case "a":
 			aws(v)
 		case "s":
 			sshKeysMenu(v)
 		case "v":
 			variables(v)
+		case "d":
+			dur := readValue("Duration (in minutes): ")
+			duration, err := strconv.Atoi(dur)
+			if err != nil {
+				fmt.Fprintln(os.Stderr, err)
+				os.Exit(1)
+			}
+			v.Duration = int64(duration) * 60
```

Comment:
Perhaps we should enforce the extents of the duration? (only allow the duration to match STS' requirements 15m <= duration <= 36h)